app/assets/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/orthogonal_bucket_picker.js

```diff
@@ -296,24 +131,27 @@ function addNecessaryBucketsToPriorityQueueOrthogonal(
           bucketAddress[v] = y;
           bucketAddress[w] += wSliceOffset;
 
-          const bucket = cube.getOrCreateBucket(bucketAddress);
           const isExtraBucket =
             y === extraYBucketStart ||
             y === extraYBucketEnd ||
             x === extraXBucketStart ||
             x === extraXBucketEnd;
 
-          if (bucket.type !== "null") {
```

Comment:
So the slight disadvantage of the new method is that buckets outside of the bounding box count against the "quota", because we no longer know in this spot whether they are null buckets or not?

Reply:
Absolutely right! However, the previous approach couldn't take into account "out of bbox"-buckets, either, since it calculated the worst-case amount. Also, the average use case shouldn't have any "out of bbox"-buckets, so, the approach should be fine.